MapboxNavigation/NavigationMapView.swift

```diff
@@ -142,20 +149,25 @@ open class NavigationMapView: MGLMapView {
             addAnnotation(destination)
         }
         
-        let source = navigationMapDelegate?.navigationMapView?(self, shapeFor: routeProgress.remainingWaypoints) ?? shape(for: Array(routeProgress.remainingWaypoints.dropLast()))
+        let source = navigationMapDelegate?.navigationMapView?(self, shapesFor: routeProgress.remainingWaypoints) ?? shapes(for: Array(routeProgress.remainingWaypoints.dropLast()))
```

Comment:
This isn’t what I meant by https://github.com/mapbox/mapbox-navigation-ios/pull/270#discussion_r132097275. The line below says `let s`. A single-letter variable should be a red flag. It was previously called `source`, but you changed it to `s` in order for this variable to be called `source`. Instead, call them each `<something>Source`.